fix bug of compression for support chatglm,chatglm2 when use `--load-8bit` (#2021)
lm-sys/FastChat: fastchat/llm_judge/README.md

This Colab [notebook](https://colab.research.google.com/drive/1ctgygDRJhVGUJTQy8-bRZCl1WNcT8De6?usp=sharing) shows how to compute the agreement between humans and GPT-4 judge with the dataset. Our results show that humans and GPT-4 judge achieve over 80\% agreement, the same level of agreement between humans.

## Citation

> 
> If you find the code or datasets helpful for your study, please consider citing the our [paper](https://arxiv.org/abs/2306.05685): "Judging LLM-as-a-judge with MT-Bench and Chatbot Arena":
> 

Reviewer: ```suggestion
Please cite the following paper if you find the code or datasets helpful.
```
